server-phoenix/priv/static/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "application.js": "/js/application-892437d2cec3f4a6041a.js",
+  "application.js.map": "/js/application-892437d2cec3f4a6041a.js.map",
+  "entrypoints": {
+    "application": {
+      "js": [
+        "/js/application-892437d2cec3f4a6041a.js"
+      ],
+      "js.map": [
+        "/js/application-892437d2cec3f4a6041a.js.map"
+      ]
+    }
+  },
+  "media/icons/icon-1-cloud.svg": "/media/icons/icon-1-cloud-43f0d5da.svg",
+  "media/icons/icon-17-cloud-snowflakes.svg": "/media/icons/icon-17-cloud-snowflakes-346f11db.svg",
+  "media/icons/icon-2-cloud-sun.svg": "/media/icons/icon-2-cloud-sun-708a1dd1.svg",
+  "media/icons/icon-24-cloud-fog.svg": "/media/icons/icon-24-cloud-fog-a67ba9b1.svg",
+  "media/icons/icon-28-moon.svg": "/media/icons/icon-28-moon-4a450174.svg",
+  "media/icons/icon-29-sun.svg": "/media/icons/icon-29-sun-e6d4cbaf.svg",
+  "media/icons/icon-3-cloud-moon.svg": "/media/icons/icon-3-cloud-moon-bb23b6f5.svg",
+  "media/icons/icon-4-cloud-rain.svg": "/media/icons/icon-4-cloud-rain-d8b5e226.svg",
+  "media/icons/icon-43-wind.svg": "/media/icons/icon-43-wind-246c5166.svg",
+  "media/icons/icon-7-cloud-snow.svg": "/media/icons/icon-7-cloud-snow-f2551726.svg",
+  "media/icons/icon-gift.svg": "/media/icons/icon-gift-9b1ee88a.svg",
+  "media/icons/icon-key.svg": "/media/icons/icon-key-2fa9212c.svg",
+  "media/icons/icon-like.svg": "/media/icons/icon-like-66fa72ef.svg",
+  "media/icons/icon-retweet.svg": "/media/icons/icon-retweet-c4b8b6bb.svg",
+  "media/icons/icon-star.svg": "/media/icons/icon-star-4ea914b7.svg",
+  "media/icons/icon-user-plus.svg": "/media/icons/icon-user-plus-9af84c57.svg",
+  "media/icons/icon-users.svg": "/media/icons/icon-users-f012ba9f.svg",
+  "media/icons/icon-wifi.svg": "/media/icons/icon-wifi-c19815f3.svg",
+  "media/images/boulder_buildings.svg": "/media/images/boulder_buildings-414f4d18.svg",
+  "media/images/buildings.png": "/media/images/buildings-026acd73.png",
+  "media/images/commit.png": "/media/images/commit-2c662617.png",
+  "media/images/displayCommit.png": "/media/images/displayCommit-8a84c3c9.png",
+  "media/images/displayPR.png": "/media/images/displayPR-ca202dda.png",
+  "media/images/displaySlack.png": "/media/images/displaySlack-62d96504.png",
+  "media/images/logo.png": "/media/images/logo-612932ca.png",
+  "media/images/pr.png": "/media/images/pr-e808acd6.png",
+  "media/images/raincloud.png": "/media/images/raincloud-f4e49358.png",
+  "media/images/slack.png": "/media/images/slack-1765e252.png",
+  "media/images/sunmoon.png": "/media/images/sunmoon-0ecf3389.png"
+}
```

Comment:
Looks like this got committed by mistake. My mistake for not adding it to `.gitignore`